images/hub/jupyterhub_config.py

```diff
@@ -67,7 +65,12 @@
     ('node_selector', 'node-selector'),
 ):
     set_config_if_not_none(c.KubeSpawner, trait, 'singleuser.' + cfg_key)
+c.KubeSpawner.storage_extra_labels = get_config('singleuser.storage-extra-labels', {})
+c.KubeSpawner.storage_extra_labels.update({
+    "hub.jupyter.org/storage-kind": "user"
```

Comment:
If there's a default label volumes should have, I think this is an issue for KubeSpawner *itself*, not this chart.

Reply:
Deleting this label as well!